packages/manager/modules/common-translations/public/translations/upload/Messages_fr_FR.json

```
@@ -0,0 +1,15 @@
+{
+  "drag_and_drop_file": "Glisser-déposer un fichier",
+  "drag_and_drop_files": "Glisser-déposer des fichiers",
+  "browse_files": "Parcourir les fichiers",
+  "max_size": "Taille maximale autorisée :",
```

Why not use the same logic for all the translations ?
Moreover, spacing after `:` differs depending on the language
```suggestion
  "max_size": "Taille maximale autorisée : {{maxSize}}",
```

---

Because the FileUpload component automatically displays both the file size and the maximum number of files when passed as props